src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+function cloneTable(index, index2) {
```

Comment:
`index, index2` are not the best names
variable name should explain what is it (e.g `rowIndexToClone`)